design-documents/graph-ql/coverage/product-reviews.graphqls

```diff
@@ -45,13 +48,13 @@ type ProductReviewRatingsMetadata {
 
 type ProductReviewRatingMetadata {
     id: String! @doc(description: "Base 64 encoded rating id.")
-    name: String! @doc(description: "The review rating name for example quality, price")
+    name: String! @doc(description: "The review rating name. For example Quality, Price")
     values: [ProductReviewRatingValueMetadata!]! @doc(description: "List of product review ratings sorted based on position.")
 }
 
 type ProductReviewRatingValueMetadata {
     value_id: String! @doc(description: "Base 64 encoded rating value id.")
-    value: String! @doc(description: "e.g Good, Perfect, 3, 4, 5")
+    value: Int!@doc(description: "The rating value. Possible values: 1, 2, 3, 4, 5")
```

Comment:
Based on https://github.com/magento/architecture/pull/343#discussion_r356165700 it should be reverted